src/app/heading/heading.component.html

```diff
@@ -6,13 +6,14 @@
     <div class="navigation__background">&nbsp;</div>
     <nav class="navigation__nav">
         <ul class="navigation__list">
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>01</span>Basement</a></li>
-            <li class="navigation__item"><a href="https://rickrolled.com/" class="navigation__link"><span>02</span>Community</a></li>
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>03</span>About</a></li>
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>04</span>placeholder</a></li>
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>05</span>placeholder</a></li>
+            <li class="navigation__item"><a routerLink="/map" class="navigation__link"><span>01</span>Basement</a></li>
```

Comment:
This looks great! These routes are going to be extremely useful as the project scales.